xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.h

```diff
@@ -129,6 +123,12 @@ class CProcessorHD : public ID3DResource
    * \return the input and output color spaces
    */
   ProcColorSpaces CalculateDXGIColorSpaces(const DXGIColorSpaceArgs& csArgs) const;
+  static DXGI_COLOR_SPACE_TYPE GetDXGIColorSpaceSource(const DXGIColorSpaceArgs& csArgs,
+                                                       bool supportHDR,
+                                                       bool supportHLG);
+  DXGI_COLOR_SPACE_TYPE GetDXGIColorSpaceTarget(const DXGIColorSpaceArgs& csArgs,
```

Comment:
not static anymore? I think it's good it doesn't rely on class members.

Reply:
Removed static as seems a bit confusing due calls others methods inside. In the other hand `GetDXGIColorSpaceSource` is self contained as not calls others methods. But actually the two methods could be static or not.